examples/src/main/java/com/opengamma/strata/examples/finance/CalibrationCheckExample.java

```diff
@@ -203,7 +208,7 @@ public static void performance_calibration_pricing() {
       for (CurveNode node : nodes) {
         if (!(node instanceof IborFixingDepositCurveNode)) {
           // IborFixingDeposit is not a real trade, so there is no appropriate comparison
-          trades.add(node.trade(VALUATION_DATE, ObservableValues.ofIdMap(quotes)));
+          trades.add(node.trade(VALUATION_DATE, MarketData.builder().addValues(quotesByKey).build()));
```

Comment:
Since `MarketDataBuilder` is a concrete class, it would be worth adding a `addValuesById()` method to simplify user code like this.